docs/content/configuration/coordinator.md

```diff
@@ -22,6 +22,7 @@ The coordinator node uses several of the global configs in [Configuration](../co
 
 |Property|Description|Default|
 |--------|-----------|-------|
+|`druid.coordinator.beOverlord`|Boolean value for whether this coordinator node should act like an overlord as well. This configuration allows users to simplify a druid cluster by not having to deploy any standalone overlord nodes. If set to true, then make sure `druid.selectors.indexing.serviceName` is correctly set to coordinator service name everywhere and all the overlord configuration must be added to the coordinator node configuration.|true|
```

Comment:
Btw, that makes me think: could you please update indexing-service.md docs too? They say a lot of "this property must be set on overlord" etc etc. If beOverlord is true and they're combined, the properties actually need to be set on the coordinator. Some language clarifying that would be helpful since without it, there is potential for users to be even more confused than they are now!